clusterman/autoscaler/pool_manager.py

```diff
@@ -236,13 +236,25 @@ def _constrain_target_capacity(
         """ Signals can return arbitrary values, so make sure we don't add or remove too much capacity """
 
         requested_delta = requested_target_capacity - self.target_capacity
+
+        # first, determine whether or not the delta is actually positive or negative.
+        # for example, if the current target capacity is above the maximum, and the
+        # requested delta is positive, the actual delta will negative, and should be, so
```

Comment:
can you fix this comment?  something seems off here (missing word maybe?)

Reply:
I've made it clearer.